cpp/ql/test/library-tests/ir/ir/raw_sanity.expected

```diff
@@ -4,6 +4,10 @@ duplicateOperand
 missingPhiOperand
 missingOperandType
 instructionWithoutSuccessor
+| ir.cpp:1002:13:1002:13 | Load: a |
+| ir.cpp:1002:23:1002:23 | Load: b |
+| ir.cpp:1002:33:1002:33 | Load: c |
+| ir.cpp:1002:43:1002:43 | Load: d |
```

Comment:
It's not great to have sanity check failures.

Reply:
Those are the child expressions from the asm statement; we can't differentiate between input and output operands without extractor changes, so I'll exclude those from IR generation for now and file a ticket with the extractor team.